internal/metrics/metricsManager_test.go

```diff
@@ -14,8 +14,8 @@ import (
 )
 
 var _ = DescribeTable(
-	"Adds agent version label to host installation phase seconds metric",
-	func(agentVersion string) {
+	"Tests that label has been added as expected",
```

Comment:
This test is not extremely useful as it stands, but it is a starting point for testing metrics.
As the previous tests was not needed anymore, I think it's best to leave a simple test to make it easier for contributors to add new tests

Reply:
I agree, looks good to me.